lib/selinux/teleport_ssh.fc.tmpl

```diff
@@ -0,0 +1,7 @@
+{{ .InstallDir }}/versions/[0-9\.]+/bin/teleport  -- gen_context(system_u:object_r:teleport_ssh_exec_t,s0)
+{{ .InstallDir }}/system/bin/teleport             -- gen_context(system_u:object_r:teleport_ssh_exec_t,s0)
+/usr/local/bin/teleport                           -- gen_context(system_u:object_r:teleport_ssh_exec_t,s0)
```

Comment:
Auto updates recently changed how Teleport is installed and now binaries are sometimes in different directories. Does this account for the change?

Reply:
Yep, the first line handles that actually. The second line handles installs from an RPM package